frontend/src/pages/Traces/TraceFlameGraph.tsx

```diff
@@ -124,10 +124,10 @@ export const TraceFlameGraph: React.FC = () => {
 					(svgContainerRef.current?.scrollLeft ?? 0) *
 						(newZoom / prevZoom)
 
-				setTimeout(() => {
+				requestAnimationFrame(() => {
```

Comment:
seems like if i scroll with mouse wheel too quickly, the graph still jumps to a random spot
https://www.loom.com/share/7b19c395b1824d6d8fd850a5dcda8b86

Reply:
I think I got this sorted out in https://github.com/highlight/highlight/pull/7663/commits/4e9987d13d33cec0a7444bbd2d3bb934707b166d! Also added support for scrolling sideways while holding the ctrl key and improved alignment of the spans when using the buttons to zoom in and out. Will merge once it's green!